dask_expr/expr.py

```diff
@@ -679,11 +694,16 @@ def _task(self, index: int):
         -------
         task: tuple
         """
-        args = tuple(self._blockwise_arg(op, index) for op in self.operands)
+        args = [self._blockwise_arg(op, index) for op in self._args]
         if self._kwargs:
-            return (apply, self.operation, args, self._kwargs)
+            return (
+                apply,
+                self.operation,
+                (tuple, list(args)),
```

Comment:
```suggestion
                args,
```
Pretty sure we can just pass the list here?

Reply:
No, this was a bug previously. The dask_dependency solving does not work when you pass a list here. I took this workaround from dask/dask

Reply:
Ah, thx. That makes sense. Changed